src/Build/Microsoft.Build.csproj

```
@@ -159,6 +159,8 @@
     <Compile Include="BackEnd\Components\SdkResolution\SdkResolverException.cs" />
     <Compile Include="BackEnd\Components\SdkResolution\TranslationHelpers.cs" />
     <Compile Include="FileSystem\*.cs" />
+    <Compile Include="Logging\FancyLogger\ANSIBuilder.cs" />
+    <Compile Include="Logging\FancyLogger\FancyLogger.cs" />
```

Our project files are an interesting mess because of a combination of legacy and our need to include different files in different TargetFrameworks. However, for a new folder, we can use the easy approach and glob in the whole folder:

```suggestion
    <Compile Include="Logging\FancyLogger\*.cs" />
```